docs/UserGuide.md

```diff
@@ -459,6 +463,7 @@ Format: `add -p PATIENT_INDEX d/TASK_DESCRIPTION | <DATE TIME> | <INTERVAL TIME_
 
 * Adds a task to a patient at the specified `PATIENT_INDEX`.
 * `DATE TIME` and `INTERVAL TIME_PERIOD` must follow the criteria defined in [Task parameters](#task-parameters).
```

Comment:
Have a look at how notes about the command are formatted in the other sections. We are using "info" and "tip" boxes.